### DISCUSSION TITLE
Proposing to refer to it as "filter evaluation" instead

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#5876] [Ideas]
It took quite some time before I connected why `{{{ this }}}` is called "filtered transclusion". And if you don't know the term, it is tricky to find the docs about it. So even if "filtered transclusion" is technically correct, I'd say **filter evaluation** is a more descriptive and intuitive term. How about referring to it as such instead, also in the docs?

**Comment by pmario** [comment]
Imo it's only a filtered evaluation if you use it with a variable eg: `list={{{xxxx}}}` if you use it in wikitext it is a "filtered transclusion", which creates a list of links. eg: ``{{{ [tag[HelloThere]] }}}`

**Reply by AnthonyMuscio** [reply]
Also a filtered transclusion can be given a template `{{{ [tag[HelloThere]] ||template}}}` 
try this in the template tiddler  `<<transclusion>> <<storyTiddler>> <br>`

**Comment by AnthonyMuscio** [comment]
Mat, 

I think it would be fair to say filtered transclusions also evaluate filters, or as you say do "filter evaluation".

However since you can set the template and its contents are transcluded "filtered transclusion" is also valid, however this is not discussed or used so often as far as I can see and the "filter evaluation" more common.

However the result of a "filtered transclusion" without a template is similar to the list widget without a template, it is strait to the screen.

Unless we can find a name that covers both uses?

Perhaps it is the Short form of "[evaluate] filters AND transclusions"

Tony
